docs/contributing/setting-up-your-local-dev-environment.md

```diff
@@ -35,7 +35,7 @@ Yarn is a package manager for your code, similar to [NPM](https://www.npmjs.com/
 - For each of your Gatsby test sites, run the `gatsby-dev` command inside the test site's directory to copy
   the built files from your cloned copy of Gatsby. It'll watch for your changes
   to Gatsby packages and copy them into the site. For more detailed instructions
-  see the [gatsby-dev-cli README](/packages/gatsby-dev-cli/) and check out the [gatsby-dev-cli demo video](https://www.youtube.com/watch?v=D0SwX1MSuas).
+  see the [gatsby-dev-cli README](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-dev-cli) and check out the [gatsby-dev-cli demo video](https://www.youtube.com/watch?v=D0SwX1MSuas).
```

Comment:
maybe better link to https://www.npmjs.com/package/gatsby-dev-cli ?

Reply:
```suggestion
  see the [gatsby-dev-cli README](https://www.npmjs.com/package/gatsby-dev-cli) and check out the [gatsby-dev-cli demo video](https://www.youtube.com/watch?v=D0SwX1MSuas).
```

Reply:
Updated, thanks.